### ISSUE TITLE
CSS beautify changing symantics

### ISSUE THREAD

**Issue by davetclark** [beautify-web/js-beautify#883]
I can't use css beautify because of this bug.

```
div [attr] :not(.class) {
    color: red;
}
```
Becomes
```
div [attr]:not(.class) {
    color: red;
}
```

Note the lack of space between ] and :

This is my config
```
{
    "_comment": "Don't edit this file directly. Changes will be overwritten",
    "preserve_newlines": true,
    "jslint_happy": true,
    "keep_array_indentation": false,
    "space_before_conditional": true,
    "max_preserve_newlines": 10,
    "brace_style": "collapse",
    "keep_function_indentation": false,
    "break_chained_methods": false,
    "eval_code": false,
    "unescape_strings": false,
    "wrap_line_length": 0,
    "git_happy": false,
    "end_with_newline": true
}
```

**Comment by chriscoyier** [comment]
Confirmed on my side too. This was [reported to us at CodePen here](https://github.com/CodePen/CodePen-Bug-Reporting-Only/issues/219).

**Comment by bitwiseman** [comment]
This will be fixed in 1.6.4, see #985.
